instaclustr/api_client_kafka_unit_test.go

```diff
@@ -0,0 +1,63 @@
+package instaclustr
+
+import (
+	"fmt"
+	"io/ioutil"
+	"testing"
+)
+
+func TestAPIClientCreateKafkaTopic(t *testing.T) {
+	filename := "data/valid_kafka_topic.json"
+	jsonStr, err := ioutil.ReadFile(filename)
+	if err != nil {
+		t.Fatalf("Failed to load %s: %s", filename, err)
+	}
+	client := SetupMock(t, "should-be-uuid/kafka/topics", `{"id":"should-be-uuid"}`, 201)
```

Comment:
what is this `should-be-uuid` thing? I suppose I haven't used SetupMock before, so I'm just not sure what this is doing

Reply:
You can imagine should-be-uuid as a cluster-id here.